cmake/Modules/dotnet/UseCSharpProjectBuilder.cmake

```diff
@@ -177,6 +178,14 @@ function(csharp_add_project name)
         DEPENDS ${sources_dep}
     )
 
+    if (${_csharp_add_project_TEST_DLL})
```

Comment:
This should be part of `FindDotNETExtra.cmake`, none of the code in `UseCSharpProjectBuilder.cmake` is dotnet-specific

Reply:
Ok. Changed